lua/telescope/_extensions/live_grep_args.lua

```diff
@@ -0,0 +1,67 @@
+-- SPDX-FileCopyrightText: 2021 Michael Weimann <mail@michael-weimann.eu>
+--
+-- SPDX-License-Identifier: MIT
+
+local prompt_parser = require("telescope-live-grep-args.prompt_parser")
+
+local telescope = require("telescope")
+local pickers = require "telescope.pickers"
+local sorters = require('telescope.sorters')
+local conf = require('telescope.config').values
+local make_entry = require('telescope.make_entry')
+local finders = require "telescope.finders"
+
+local tbl_clone = function(original)
+  local copy = {}
+  for key, value in pairs(original) do
+    copy[key] = value
+  end
+  return copy
+end
+
+local setup_opts = {}
+local prompt_title = "live grep (args)"
+
+local live_grep_args = function(opts)
+  opts = vim.tbl_extend('force', setup_opts, opts or {})
+
+  opts.vimgrep_arguments = opts.vimgrep_arguments or conf.vimgrep_arguments
+  opts.entry_maker = opts.entry_maker or make_entry.gen_from_vimgrep(opts)
+  opts.cwd = opts.cwd and vim.fn.expand(opts.cwd)
+
+  local cmd_generator = function(prompt)
+    if not prompt or prompt == "" then
+      return nil
+    end
+
+    local args = tbl_clone(opts.vimgrep_arguments)
+    local prompt_parts = prompt_parser.parse(prompt, opts.auto_quoting)
+
+    local cmd = vim.tbl_flatten { args, prompt_parts }
+    return cmd
+  end
+
+  pickers.new(opts, {
+    prompt_title = prompt_title,
+    finder = finders.new_job(cmd_generator, opts.entry_maker, opts.max_results, opts.cwd),
+    previewer = conf.grep_previewer(opts),
+    sorter = sorters.highlighter_only(opts),
+  }):find()
+end
+
+local live_grep_raw = function(opts)
+  prompt_title = "live_grep_raw -> live_grep_args: migrate your config!"
```

Comment:
can we just print a message? Or use `vim.notify` to make this more explicit what the user needs to do?

Reply:
Done. I found a way so that users will hopefully actually see it :smile: 

https://github.com/nvim-telescope/telescope-rg.nvim/pull/18/files#diff-11de1691738d4464ce9eb24b3802d76daddd2201329fb8276f2f99903eb9dc60R7